Source/JavaScriptCore/wasm/WasmExceptionType.h

```diff
@@ -72,7 +72,9 @@ namespace Wasm {
     macro(NullRefAsNonNull, "ref.as_non_null to a null reference"_s) \
     macro(CastFailure, "ref.cast failed to cast reference to target heap type"_s) \
     macro(OutOfBoundsDataSegmentAccess, "Offset + array length would exceed the size of a data segment"_s) \
-    macro(OutOfBoundsElementSegmentAccess, "Offset + array length would exceed the length of an element segment"_s)
+    macro(OutOfBoundsElementSegmentAccess, "Offset + array length would exceed the length of an element segment"_s) \
+    macro(OutOfMemory,  "Out of memory"_s) \
+    macro(ParseError,  "Parse error"_s)
```

Comment:
I don't think you need this Error type. We shouldn't be parsing code (that won't parse) after the module is instantiated.